code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -161,6 +161,23 @@
 	if(wear_id)
 		msg += "[p_they(TRUE)] [p_are()] wearing [bicon(wear_id)] \a [wear_id].\n"
 
+	//Stamp Marks
+	var/numberOfStampTypes = length(stamp_marks_desc)
+	if(numberOfStampTypes)
+		msg += "[p_they(TRUE)] [p_have()] been stamped: "
+		var/stampTypeCounter = 0
+
+		for(var/i in stamp_marks_desc)
```

Comment:
Nit: `i` no longer makes mnemonic sense, given that it's not an `image` anymore (nor is it a loop counter). Consider renaming it to `var/desc` or something